pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -38,6 +38,7 @@ const (
 
 var (
 	openShiftContainerBinds = []string{
+		"/var/log:/var/log:rw",
```

Comment:
For aggregated logging to work, fluentd relies on the node to write symlinks in /var/log/containers that serve as metadata to the actual container logs. Without mounting /var/log the containerized node would be writing symlinks to its container filesystem where fluentd can't read them.